CRMWebResourceUpload/Program.cs

```diff
@@ -190,7 +189,7 @@ static List<string> GetLocalFiles(string path, string fullPath, bool firstRun =
             List<string> output = new List<string>();
             foreach (var i in localFiles)
             {
-                output.Add(i.Replace(fullPath + "\\", ""));
+                output.Add(i.Replace(fullPath + "\\", "").Replace("\\", "/"));

```

Comment:
Why do you need to replace the directory slashes here?  Would you expect to run this program in a non-Windows environment?

Reply:
On the web everything uses forward slashes and windows handles forward slashes in name just fine and I think there was something also that I needed to handle this but I don't remember what it was at this point in time.